.github/ISSUE_TEMPLATE/documentation_request_new.yml

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Documentation - New Documentation Request
```

Comment:
Can we just have one documentation form, "Report missing or incorrect documentation"? That seems to cover "request new".

Reply:
We can do that, but the fields and descriptions will be a little wonky since they need to umbrella cover new or fixes. The current templates get around this by having the user delete the part they don't want but that's not an option with forms. 

I'll try some stuff out and we can see how a single form would look.